examples/FUOTA/UpdateServer/MQTT/ChirpStackMQTT.py

```
@@ -0,0 +1,45 @@
+    def __on_MQTT_Message(self, Client, UserData, Message):
+        self.__Message = Message.payload.decode("ascii")
+        self.__Logger.debug("Receive message with topic {} and payload {}".format(Message.topic, self.__Message))
+
+    def SubscribeApplication(self, AppID):
+        Topic = "application/{}/device/+/event/up".format(AppID)
+
+        self.__Logger.debug("Subscribe to topic {}".format(Topic))
+        self.__Client.subscribe(Topic)
+
+    def WaitForMessage(self):
+        while(self.__Message == None):
```

Ruff (0.14.3)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)